### DISCUSSION TITLE
cargo plugin

### DISCUSSION THREAD

**Discussion by drahnr** [rpm-rs/rpm#74] [General]
Currently `cargo-rpm` is deprecated, and it might make sense to maintain an integration similar to `cargo-generate-rpm` under that name.

Iff there is consensus, we might want to ask https://github.com/tony-iqlusion for ownership transfer

**Comment by dralley** [comment]
What are the current differences between `cargo-rpm` and `cargo-generate-rpm` and what reasons exist to pick one over another?  `cargo-generate-rpm` seems to be already maintained and already uses `rpm-rs`.

Generally if there's an existing tool that works and is maintained (or is relatively bug-free even without maintainence) I'm not super pumped to replace it.

**Reply by dralley** [reply]
We could ask if they'd be interested in joining the org, but if I were in their place I probably wouldn't feel a strong reason to unless I knew it I wouldn't have time to maintain it.

**Reply by drahnr** [reply]
I think asking for integration is a good idea.

Not sure if you saw, it migrated to rpm-rs/rpm already

**Reply by dralley** [reply]
Summarizing the differences:

* `cargo-rpm` - cargo plugin that generates spec files and provides a wrapper around rpmbuild for building packages
* `cargo-generate-rpm` - cargo plugin that generates RPM packages using this library, no spec file
* `rust2rpm` - generates spec files from crates for use with distro packaging, not a cargo plugin, also written in Python not Rust

**Reply by dralley** [reply]
What do you mean by integration?  Inviting them to join the org?  I don't have an issue with that

Otherwise, as you say they're already integrated since they're using it
